scripts/templates/grafana-dashboards/dm-apps.json

```diff
@@ -0,0 +1,747 @@
+{
+  "title": "DM-Apps",
+  "tags": [],
+  "style": "dark",
+  "timezone": "browser",
+  "editable": true,
```

Comment:
Does it? For me it removes the "Edit" buttons from the graphs. I can still drag things around on the dashboard but when I leave it doesn't offer to save any changes